bin/cron/upload_to_firebase

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env ruby
+
+# This script fetches weather forecast data from api.openweathermap and uploads to the shared firebase channel.
+require_relative '../../deployment'
+require 'cdo/only_one'
+require 'net/http'
+require 'json'
+require_relative '../../shared/middleware/helpers/firebase_helper'
+
+WeatherForecastOffice = Struct.new(:city, :state, :zip)
+WeatherRecord = Struct.new(:id, :city, :state, :timestamp, :temp_min, :temp_max, :weather, :icon, :wind_speed)
+
+def get_weather_data
+  forecast_offices = [
+    WeatherForecastOffice.new("Birmingham", "Alabama", "35040"),
+    WeatherForecastOffice.new("Eureka", "California", "95501"),
+    WeatherForecastOffice.new("Miami", "Florida", "33165"),
+    WeatherForecastOffice.new("St. Louis", "Missouri", "63304"),
+    WeatherForecastOffice.new("Wakefield", "Virginia", "23888")
+  ]
+  records = []
+  id = 1
+  forecast_offices.each do |office|
+    puts office
+    url = "http://api.openweathermap.org/data/2.5/forecast?zip=#{office.zip},us&units=imperial&appid=c719e74bc77b787d940386405e0cbb83"
```

Comment:
It might be worth looking into whether there is a manifest where we keep URLs like this

Reply:
Capturing offline discussion- going to leave this URL here so it's easy to see where the data is coming from. We can always move it somewhere else later if desired